source-code/website/src/pages/editor/state.ts

```diff
@@ -177,7 +177,20 @@ async function cloneRepository(args: {
 	});
 	// triggering a side effect here to trigger a re-render
 	// of components that depends on fs
+
 	const date = new Date();
+	//checked the existing branches and added a "inlang" translation branch, if this branch does not exist
+	let branches = await raw.listBranches({ fs, dir: "/", remote: "origin" });
+	if (branches.includes("inlang") == false) {
```

Comment:
always use triple equality `===`! using double equality is a footgun. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#:~:text=Triple%20equals%20(%20%3D%3D%3D%20)%20will%20do,types%20differ%2C%20false%20is%20returned.